Sources/Playbook/Components/Icon Value/IconValueCatalog.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Playbook Swift Design System
+//
+//  Copyright © 2024 Power Home Remodeling Group
+//  This software is distributed under the ISC License
+//
+//  IconValueCatalog.swift
+//
+
+import SwiftUI
+
+public struct IconValueCatalog: View {
+    public var body: some View {
+      ScrollView {
```

Comment:
What about using our DocStack?

Reply:
It's not available in main yet. or it wasn't when I started this